contracts/token/ERC721/IERC721.sol

```diff
@@ -81,7 +81,9 @@ interface IERC721 is IERC165 {
     /**
      * @dev Transfers `tokenId` token from `from` to `to`.
      *
-     * WARNING: Usage of this method is discouraged, use {safeTransferFrom} whenever possible.
+     * WARNING: Note that the caller is responsible to confirm that the recipient is capable of receiving ERC721
+     * or else they may be permanently lost. Usage of {safeTransferFrom} is encouraged, though the caller must understand
+     * this adds an external call to {transferFrom}, potentially creating a reentrancy vulnerability.
```

Comment:
Current wording seems to imply that there is a call to `transferFrom`.

```suggestion
     * this adds an external call which potentially creating a reentrancy vulnerability.
```

Reply:
You're right! I'll commit the changes addressing your feedback :)